recipes/betterproto/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "betterproto" %}
+{% set version = "1.2.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/betterproto-{{ version }}.tar.gz
+  sha256: 74a3ab34646054f674d236d1229ba8182dc2eae86feb249b8590ef496ce9803d
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  entry_points:
+    - protoc-gen-python_betterproto=betterproto.plugin:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - backports-datetime-fromisoformat  # [py<37]
+    - dataclasses  # [py<37]
+    - grpclib
+    - python
+    - stringcase
+
+test:
+  imports:
+    - betterproto
+  commands:
+    - pip check
+  requires:
+    - pip
+    - protobuf
```

Comment:
Why do you require `protobuf` in the tests but not in run?

Reply:
Good catch, thanks. This was a leftover from a more expanded set of manual tests against the protobuf generators, as I hadn't used them via `conda` before, and is no longer needed.